index.bs

```diff
@@ -209,14 +209,16 @@ the user agent MUST run the following steps:
 8. Set {{pictureInPictureElement}} to |video|.
 9. Let <dfn>Picture-in-Picture window</dfn> be a new instance of
     {{PictureInPictureWindow}} associated with {{pictureInPictureElement}}.
-10. Append <a>relevant settings object</a>'s <a>origin</a> to
+10. When the Picture-in-Picture window is opened it MUST be visible, even when
```

Comment:
I'm not sure this is the right place actually.

Reply:
How about simply moving the following at the end of `## Request Picture-in-Picture ## {#request-pip}` section?

```
When {{pictureInPictureElement}} is set, the Picture-in-Picture window MUST
be visible, even when the <a>Document</a> is not in focus or hidden. The user
agent SHOULD provide a way for users to manually close the Picture-in-Picture
window.
```

Reply:
Yes, that would work. I was in two minds about it when I made this edit...

Reply:
I updated it as you suggested.